src/global_distrib/mod_global_distrib_mapping_redis.erl

```diff
@@ -279,15 +285,17 @@ set_endpoints(Endpoints) ->
     refresh_set(EndpointsKey, BinEndpoints).
 
 -spec endpoint_to_binary(mod_global_distrib_utils:endpoint()) -> binary().
```

Comment:
Why not `IP:PORT` format instead of `IP#PORT`, which is almost standard way to specify addresses?

Reply:
Good question! But wrong place to pose it! :D We could change it but this PR does not try to do it.

Reply:
resolved for this PR. will discuss and create a redmine task.

Reply:
Simple answer: With `:` it would be less straightforward to parse binary with IPv6+Port. With `#` it is as easy as it can be. :)

Reply:
Because a serialized representation of `IP#PORT` requires much less deserialization work when IPv6 is a factor.

Anyway, this is an internal representation of data that is human readable mostly because I fancied it rather than anything else. Does this really warrant a task? Why?

Reply:
@kzemek We need a tech meeting to discuss whether we want the separator to be 😂 or 😍

Reply:
Why not `{"ip": "127.0.0.1", "port": 6666}` format? :trollface:

Reply:
That's hard to parse too

Reply:
Agree

```erlang
jiffy:decode(<<"{\"ip\": \"127.0.0.1\", \"port\": 6666}">>, [return_maps]).
#{<<"ip">> => <<"127.0.0.1">>,<<"port">> => 6666}
```